### PR TITLE
Add content/pt-br/application_programming_interface.md

### PR DESCRIPTION
**Pull request by jessicalins**
Reopening - same content as https://github.com/cncf/glossary/pull/273, but now with an organized tree/commits. Sorry for the confusion :sweat_smile: 

Signed-off-by: Jéssica Lins <jessicaalins@gmail.com>

### CODE REVIEW

**Comment by jessicalins** [comment]
cc @brunoguidone

File: content/pt-br/application_programming_interface.md
Comment (edsoncelio) [inline content/pt-br/application_programming_interface.md:2]: Interface de programação de aplicações to Interface de Programação de Aplicações

File: content/pt-br/application_programming_interface.md
Comment (edsoncelio) [inline content/pt-br/application_programming_interface.md:4]: Tecnologia to tecnologia

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi @jessicalins I just left some comments :)

File: content/pt-br/application_programming_interface.md
Comment (jessicalins) [inline content/pt-br/application_programming_interface.md:2]:
```suggestion
title: Interface de Programação de Aplicações
```

**Review by edsoncelio — APPROVED** [review]
LGTM

**Review by brunoguidone — APPROVED** [review]
Review ok

**Comment by edsoncelio** [comment]
@seokho-son could you help us with this? We need just one approval from a codeowner :D

**Pull request merged**
